app/javascript/miq-redux/index.ts

```diff
@@ -0,0 +1,14 @@
+import {createStore} from 'redux';
+import {addReducer, applyReducerHash} from './reducer';
```

Comment:
I fon't know why, but for some reason my VS code did not picked up .editorconfig and went on it's own with formatting. But I will definetely use tslint to be mean to everyone if they'll make any mistake.